lxd/db/certificates.go

```diff
@@ -17,9 +13,9 @@ type CertInfo struct {
 }
 
 // CertsGet returns all certificates from the DB as CertBaseInfo objects.
-func CertsGet(db *sql.DB) (certs []*CertInfo, err error) {
+func (n *Node) CertsGet() (certs []*CertInfo, err error) {
```

Comment:
A nit I didn't notice until now, but can we use "Certificates" and "Certificate" in there rather than "Certs" and "Cert"?

This would be more consistent with what we're doing in the client code and much improve my odds of finding the right thing when grepping through the code.

Reply:
Sure, done in b5cd9b60aaba97e565f79e3f1259c139ee6d8aea.